recipe/meta.yaml

```diff
@@ -19,35 +12,29 @@ source:
 build:
   number: 2
```

Comment:
should bump build number

Reply:
no, currently it does not change the content of the package